src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -38,7 +38,29 @@ export class BrowsingContextProcessor {
     );
   }
 
-  static #removeContext(contextId: string) {
+  async #removeContext(contextId: string): Promise<void> {
+    if (!BrowsingContextProcessor.#hasKnownContext(contextId)) {
+      return;
+    }
+    const context = BrowsingContextProcessor.#getKnownContext(contextId);
+    // Remove context's children.
```

Comment:
add `Context::removeChildren()` method and call it from here and frameNavigated?